index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
> Mocha runs the tests in parallel

I don't think that's the case. The problem is that Node itself uses `Object.keys`, so deleting it can cause problems. For example:

```javascript
> delete Object.keys
true
> console.log({})
TypeError: Object.keys is not a function
```

I don't have a good solution to this problem. The *simplest* option is to remove the "fast" path, but I'm open to other suggestions. :)